landing_page/static/landing_page/js/index.js

```diff
@@ -0,0 +1,9 @@
+function searchMovies(event) {
+    key = event.keyCode
+
+    if (key == 13) {
```

Comment:
can we look at the name of the key rather than 13? or, if anything, at least a comment that says "user clicked enter"

Reply:
This is fixed. There was a way to do it by name